library/errata_tool_user_info.py

```diff
@@ -0,0 +1,126 @@
+from ansible.module_utils import common_errata_tool
+from ansible.module_utils.basic import AnsibleModule
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.0",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+
+DOCUMENTATION = """
+---
+module: errata_tool_user_info
+
+short_description: Query users in the Errata Tool
+description:
+   - Query users within Red Hat's Errata Tool.
```

Comment:
Let's document that this module does not modify state, and it always queries the user information, even in check mode.